moped-data-events/activity_log/MopedEvent.py

```diff
@@ -44,6 +47,9 @@ class MopedEvent:
           }) {
             affected_rows
           }
+          update_moped_project(where: {project_id: {_eq: $recordId}}, _set: {updated_at: $timestamp}) {
+            affected_rows
+          }
```

Comment:
Brilliant, cool that you made it a timestamptz value, those are rare in GraphQL! 👍